acceptance/clients/clients.go

```diff
@@ -285,6 +285,27 @@ func NewBareMetalV1NoAuthClient() (*gophercloud.ServiceClient, error) {
 	})
 }
 
+// NewBareMetalInspectorV1Client returns a *ServiceClient for making calls
+// to the OpenStack Bare Metal Inspector v1 API. An error will be returned
+// if authentication or client creation was not possible.
+func NewBareMetalInspectorV1Client() (*gophercloud.ServiceClient, error) {
```

Comment:
Comment on the whole patch: it's either "ironic inspector" OR "baremetal introspection", but not a mix of them.

Reply:
Good catch.

Please use Baremetal / BareMetal where appropriate. Gophercloud uses service names and not project/code names.

Reply:
changed to baremetalintrospection